src/Shared/BinarySerializer.cs

```diff
@@ -3,11 +3,14 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 //
 
+using System;
 using System.IO;
-using System.Runtime.Serialization.Formatters.Binary;
+using System.Text;
+using ProtoBuf;
 
 namespace Microsoft.Web.Redis
 {
+
     public class BinarySerializer : ISerializer
     {
         public byte[] Serialize(object data)
```

Comment:
what `object data` payloads do we expect here? do we expect that all existing usages support protobuf-net?